unit-tests/run-unit-tests.py

```diff
@@ -278,6 +282,11 @@ def __init__( self, source, line_prefix ):
                     log.e( source + str(context.index) + ': device directive with no devices listed' )
                 else:
                     self._configurations.append( params )
+            elif directive == 'priority':
+                if params[0].isdigit():

```

Comment:
Does this work for other malformed lines?
E.g.,
```
#test:priority
#test:priority 1 100
#test:priority -1
```

Reply:
It will work for first and last. I'll add a length check